hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/FileServiceImpl.java

```diff
@@ -24,6 +24,7 @@
 /** Standard implementation of the {@link FileService} {@link com.hedera.node.app.spi.Service}. */
 public final class FileServiceImpl implements FileService {
     public static final String BLOBS_KEY = "FILES";
+    public static final String UPGRADE_FILE_KEY = "UPGRADE_FILE";
```

Comment:
This will replace UPGRADE_FILES_KEY in FreezeServiceImpl. I will do a follow-up PR to delete it from there.

Reply:
ok no problem